lib/rules/object-curly-newline.js

```diff
@@ -69,6 +68,24 @@ function normalizeOptionValue(value) {
     return { multiline, minProperties, consistent };
 }
 
+/**
+ * Checks if an object is a plain object
+ * @param {any} object The object to check
+ * @returns {boolean} `true` if the option is a plain object, otherwise `false`
+ */
+function isPlainObject(object) {
+    return Object.prototype.toString.call(object) === "[object Object]";
+}
```

Comment:
In the context of this rule, we just need to check if the given value is an object. Can we do something like this, to not rely on `"[object Object]"`:

```js
/**
 * Check if a value is an object.
 * @param {any} value The value to check
 * @returns {boolean} `true` if the value is an object, otherwise `false`
 */
function isObject(value) {
    return typeof value === "object" && value !== null;
}
```

Reply:
Resolved in a1bd47baa.